packages/muelu/research/regionMG/src/SetupRegionSmoothers_def.hpp

```diff
@@ -306,12 +287,12 @@ calcNorm2(Array<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> >
  */
 template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
 Scalar
-dotProd(Array<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > >& regX,
-        Array<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > >& regY,
-        const std::vector<RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > > rowImportPerGrp)
+dotProd(RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> >& regX,
+        RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> >& regY,
+        const RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > rowImportPerGrp)
```

Comment:
The removed `std::vector` was related to the concept of "groups". Hence, this could be renamed to 
```suggestion
        const RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > rowImport)
```